whipper/common/program.py

```diff
@@ -416,6 +418,12 @@ def getTagList(self, number, mbdiscid):
                     mbidTrack = track.mbid
                     mbidTrackArtist = track.mbidArtist
                     mbidWorks = track.mbidWorks
+                    if len(track.composers) == 0:
+                        track.composers = track.artist
+                    if len(track.performers) == 0:
+                        track.performers = [track.recordingArtist]
```

Comment:
These four lines should be taken out without the `classical` mode.
1. Composers are not the track artist for non-classical.
2. `composers` were made to be a list in another part of the code, so doesn’t make sense to make it a string here.

Reply:
Yeah since we are not going to use recording artist and track artist as fallback (and i made yet another mistake) so this indeed should be removed